src/pair_surround.py

```diff
@@ -0,0 +1,46 @@
+from talon import Context, Module
+
+mod = Module()
+ctx = Context()
+
+ctx.matches = r"""
+app: vscode
+"""
+
+pair_symbols = {
+    "angle": "angleBrackets",
+    "diamond": "angleBrackets",
+    "curly": "curlyBrackets",
+    "round": "parentheses",
+    "square": "squareBrackets",
+    "quad": "doubleQuotes",
+    "twin": "singleQuotes",
+}
+
+mod.list("cursorless_pair_symbol", desc="A pair symbol")
+ctx.lists["self.cursorless_pair_symbol"] = pair_symbols
+
+
+pair_surround_types = {
+    "inner": {"insideOutsideType": "inside"},
+    "outer": {"insideOutsideType": "outside"}
+}
+
+mod.list("cursorless_pair_surround_type", desc="Supported pair surround types")
+ctx.lists["self.cursorless_pair_surround_type"] = pair_surround_types.keys()
```

Comment:
"pair surround" => "inside outside"

Reply:
also, the inside-outside thing is not specific to pairs, so I'd be tempted to move it to its own file

Reply:
And make it just a top-level modifier that goes into the bag of words rather than requiring it to be just before the pair surround type